src/js/rx/components/GlossaryModal.jsx

```diff
@@ -23,28 +18,32 @@ class GlossaryModal extends React.Component {
     let element;
     if (this.props.isVisible) {
```

Comment:
This should probably be added to the propTypes, but I've also noticed before that it'd be redundant to do handling around the visibility of the modal when Modal already does that.

And looking at the propTypes of Modal, it doesn't seem like we should require the content, since most use cases have an initially hidden modal that doesn't have content.

Anyway, just a couple of thoughts on how the apps are using modals, and not something that needs to change for this PR.